diff --git a/tools/ci_build/github/android/build_aar_package.py b/tools/ci_build/github/android/build_aar_package.py
index 0000000..0000000 100644
--- a/tools/ci_build/github/android/build_aar_package.py
+++ b/tools/ci_build/github/android/build_aar_package.py
@@ -96,6 +96,6 @@ def _build_aar(args):
     base_build_command = (
         [sys.executable, BUILD_PY]
         + build_settings["build_params"]
-        + ["--config=" + build_config, "--use_vcpkg", "--use_vcpkg_ms_internal_asset_cache"]
+        + ["--config=" + build_config, "--use_vcpkg"]
     )
     header_files_path = ""
